<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockBrowser</class>
 <widget class="QWidget" name="BlockBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>浏览方块</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QSpinBox" name="sb_scale_ratio">
     <property name="suffix">
      <string> x</string>
     </property>
     <property name="prefix">
      <string>放大倍率：</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="combobox_select_blk">
     <property name="placeholderText">
      <string>选择方块</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combobox_select_face">
     <property name="placeholderText">
      <string>选择视角</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QLabel" name="label_image">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextBrowser" name="tb_view_id">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="documentTitle">
      <string>方块id</string>
     </property>
     <property name="placeholderText">
      <string>方块id</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pb_save_current_image">
     <property name="text">
      <string>保存当前图片</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
